optuna/samplers/_cmaes.py

```diff
@@ -60,7 +60,7 @@ def objective(trial):
             study.optimize(objective, n_trials=20)
 
     Please note that this sampler does not support CategoricalDistribution.
-    However, :class:`~optuna.distributions.DiscreteUniformDistribution`
+    However, :class:`~optuna.distributions.FloatDistribution` with ``step``,
```

Comment:
a nit, but `Int(Log)Distribution` at the two line below seems to be replaced with `IntDistribution`.

Reply:
That's right. I updated it in bf058c1.